.appveyor.yml

```diff
@@ -39,10 +43,8 @@ cache:
   - C:\Users\appveyor\.cargo\registry
   - target
 
-build_script:
```

Comment:
Although `cargo test` builds the project, I think we should separate build and test stages.

Reply:
What's the rationale?

Reply:
From what I understand, for certain windows systems this helps mitigate the "directory does not contain a project or solution file" error. But my source is from 3years ago.

Reply:
Because we separated them in Travis config too, whereas you merged them in Appveyor only. Also you don't have to call `cargo test --all` to test `testcrate` (with env vars), you can call it directly: `cargo test -p testcrate` IIRC.

Reply:
Yes so you would prefer:
```
cargo test -p testcrate
cargo test -p libsodium-sys
cargo test -p sodiumoxyde
```
over
```
cargo test --all
```

Reply:
And I can do the same for the travis build.